metrics/lib/common.bash

```diff
@@ -194,30 +186,21 @@ show_system_state() {
 	echo "Showing system state:"
 	echo " --Docker ps--"
 	${DOCKER_EXE} ps -a
-	echo " --${RUNTIME} list--"
-	local RPATH=$(command -v ${RUNTIME})
-	sudo ${RPATH} list
 
 	local processes="kata-proxy kata-shim kata-runtime $(basename ${HYPERVISOR_PATH} | cut -d '-' -f1)"
```

Comment:
I think you can remove `kata-proxy` process

Reply:
Also `kata-shim` is not valid, it should be `containerd-shim-kata-v2`

Reply:
Now this function is useless, I think it's safe to remove it